Cirecuit_Playground_Express_Pinata/code.py

```diff
@@ -0,0 +1,40 @@
+# Circuit Playground Express Piñata by Dano Wall for Adafruit Industries
+# CircuitPython code by Mike Barela for Adafruit Industries, MIT License
+import time
+import gc
+import random
+import board
+import pulseio
+from adafruit_motor import servo
+from adafruit_circuitplayground.express import cpx
+gc.collect()
```

Comment:
i dont believe this is necessary anymore, as of 4.x we collect after each import

Reply:
It failed to work before I added it. Shall I try again?

Reply:
what cpy version are you running?

Reply:
3.x I think on CPX

Reply:
try 4.x

Reply:
4.0.0 works w/o garbage collection. Tested